Kitodo/src/main/java/org/kitodo/production/services/data/ProcessService.java

```diff
@@ -511,6 +532,35 @@ public void refresh(Process process) {
         return findByQuery(getQueryProjectId(id), related);
     }
 
+    /**
+     * Find processes by title.
+     *
+     * @param title
+     *            the title
+     * @return a list of processes
+     * @throws DataException
+     *             when there is an error on conversion
+     */
+    public List<ProcessDTO> findByTitle(String title) throws DataException {
+        return convertJSONObjectsToDTOs(findByTitle(title, true), true);
+    }
+
+    /**
+     * Find processes by title with wildcard.
+     *
+     * @param title
+     *            the title
+     * @return a list of processes
+     * @throws DataException when there is an error on conversion
+     */
+    public List<ProcessDTO> findDTOsByTitleWithWildcard(String title) throws DataException {
+        return convertJSONObjectsToDTOs(super.findByTitleWithWildcard(title), false);
+    }
+
+    private QueryBuilder getQueryProcessTitle(String title) {
+        return createSimpleQuery(ProcessTypeField.TITLE.getKey(), title, true);
```

Comment:
getQueryTitle method already exists in TitleSearchService -> ProcessService extends TitleSearchService